packages/domains-table/src/domains-table/domains-table-row.tsx

```diff
@@ -80,10 +85,21 @@ export function DomainsTableRow( { domain }: DomainsTableRowProps ) {
 		return currentDomainData.owner.replace( / \((?!.*\().+\)$/, '' );
 	};
 
+	const handleSelect = () => {
+		window.location.href = domainManagementLink;
+	};
+
 	return (
-		<tr key={ domain.domain }>
+		<tr
+			key={ domain.domain }
+			className="domains-table__row"
+			onClick={ domainManagementLink ? handleSelect : undefined }
+		>
 			{ canSelectAnyDomains && (
-				<td className="domains-table-checkbox-td">
+				<td
+					className="domains-table-checkbox-td"
+					onClick={ ( e: MouseEvent ) => e.stopPropagation() }
```

Comment:
Ahh it looks like you already are disabling propogation on the TD, but the TD is actually not wide enough to catch those clicks.I think we can fix this by adding a fixed width to it


<img width="879" alt="Screenshot 2024-05-13 at 12 28 57 pm" src="https://github.com/Automattic/wp-calypso/assets/22446385/99043db3-c77c-4133-af89-20c2d6af3165">